### DISCUSSION TITLE
Laravel Backup Package Issue

### DISCUSSION THREAD

**Discussion by wadchathuranga** [spatie/laravel-backup#1266] [Q&A]
when I use the Laravel backup package & try to run

	Route::get('backup', function() {
			Artisan::call('backup:run', ['--only-files'=>true]);
		    return 'backup complete';
		});

it gives an error "The command "backup:run" does not exist.". Can anyone please help me?

**Comment by wadchathuranga** [comment]
@freekmurze, do you have any suggestions to fix this issue?

**Comment by HDVinnie** [comment]
Dupe of https://github.com/spatie/laravel-backup/discussions/1265 ....
